REFERENCE.md

```diff
@@ -257,3 +257,11 @@ configure your most favourite ssh key type. This will be used to connect to the
 
 Default value: `'ed25519'`
 
+##### `backuptime`
+
+Data type: `Hash[String,String]`
```

Comment:
this file is autogenerated from the comments in init.pp. Please update only the init.pp and run `bundle exec rake reference` afterwards.